Update installation.markdown

gozasc opened the pull request:
One grammatical change ("atleast" should be two separate words) and spacing/punctuation changes for the sake of consistency on the page.

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

gozasc committed:
Update installation.markdown

One grammatical change ("atleast" should be two separate words) and spacing/punctuation changes for the sake of consistency on the page.

frenck commented on source/hassio/installation.markdown:
```diff
@@ -11,8 +11,8 @@ The following will take you through the steps required to install Home Assistant
      - [Raspberry Pi 3 Model B and B+ 64-bit][pi3-64]
-     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 32-bit][pi4-32] (recommended)
-     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 64-bit][pi4-64]
+     - [Raspberry Pi 4 Model B (1GB, 2GB and 4GB model) 32-bit][pi4-32] (recommended)
+     - [Raspberry Pi 4 Model B (1GB, 2GB and 4GB model) 64-bit][pi4-64]
```
Please revert the change of removing the space (for all occurrences in this PR).

There should be a space between the value and its unit of measurement.

For reasoning and more information, see both the Microsoft & Google Style Guides on writing documentation.

- <https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/units-of-measure-terms>
- <https://developers.google.com/style/units-of-measure>

gozasc replied:
Sounds good. Thank you for letting me know about the Google/Microsoft guidelines regarding units of measure. 

If that is the case, please be sure to correct (by adding) a space to the bottom portion regarding VMware Workstation. There is a "2GB RAM" down there, which is what prompted me to recommend standardizing the spaces to begin with.

frenck replied:
Feel free to correct that as you process the review comments on your PR.

gozasc committed:
Update installation.markdown

Resubmitting with added spaces between quantities and units of measurement (e.g. "2 GB") per the Google/Microsoft technical writing guidelines. Also added a space between "atleast" to make it two words as it should be.

gozasc reviewed (commented):
Resubmitting with added spaces between quantities and units of measurement (e.g. "2 GB") per the Google/Microsoft technical writing guidelines. Also added a space between "atleast" to make it two words as it should be.

frenck commented on source/hassio/installation.markdown:
```diff
@@ -12,7 +12,7 @@ The following will take you through the steps required to install Home Assistant
      - [Raspberry Pi 3 Model B and B+ 64-bit][pi3-64]
      - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 32-bit][pi4-32] (recommended)
-     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 64-bit][pi4-64]
+     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4GB model) 64-bit][pi4-64]
```
```suggestion
     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 64-bit][pi4-64]
```

frenck commented on source/hassio/installation.markdown:
```diff
@@ -33,10 +33,10 @@ The following will take you through the steps required to install Home Assistant
    - Load the appliance image into your virtual machine software. (Note: You are free to assign as much resources as you wish to the VM, please assign enough based on your add-on needs)
-     - For VirtualBox create a new virtual machine, select "Other Linux (64Bit), assign it atleast 2 GB of memory and "Use an existing virtual hard disk file", select the VDI file from above, afterwards edit the "Settings" of the VM and go "System" then Motherboard and Enable EFI, then "Network" "Adapter 1" Bridged and your adapter
-     - For Hyper-V create a new virtual machine, select "Generation 2", assign it at least 2 GB of memory and select "Connection -> "Your Virtual Switch that is bridged", then "Use an existing virtual hard disk" and select the VHDX file from above, after creation go to "Settings" -> "Security" and deselect "Enable Secure Boot"
+     - For VirtualBox create a new virtual machine, select "Other Linux (64Bit), assign it at least 2GB of memory and "Use an existing virtual hard disk file", select the VDI file from above, afterwards edit the "Settings" of the VM and go "System" then Motherboard and Enable EFI, then "Network" "Adapter 1" Bridged and your adapter.
```
```suggestion
     - For VirtualBox create a new virtual machine, select "Other Linux (64Bit), assign it at least 2 GB of memory and "Use an existing virtual hard disk file", select the VDI file from above, afterward edit the "Settings" of the VM and go "System" then Motherboard and Enable EFI, then "Network" "Adapter 1" Bridged and your adapter.
```

gozasc committed:
Update installation.markdown

Added spaces back.

frenck reviewed (approved):
Thanks, @gozasc 👍
